hastory-server/src/Data/Hastory/Server/Utils.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Data.Hastory.Server.Utils where
+
+import Crypto.Hash (Digest, SHA256(..), hashWith)
+import qualified Data.ByteString.Char8 as C
+import Data.Hastory.Types (Entry(..), ServerEntry(..), SyncRequest(..))
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+import Network.HostName (HostName)
+
+hashEntry :: Entry -> HostName -> Digest SHA256
+hashEntry Entry {..} host = hashWith SHA256 (unifiedData :: C.ByteString)
+  where
+    unifiedData =
+      mconcat
+        [ T.encodeUtf8 entryText
+        , C.pack $ show entryWorkingDir
```

Comment:
You probably want to `T.encodeUtf8 . T.pack` instead of `C.pack` then at least the right thing is hashed.

Reply:
👍